.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+node_modules
+*.map
```

Comment:
Please add a .editorconfig file and configure your IDE to add a newline at the end.

Reply:
FWIW, I hadn't touched that file.

Reply:
Yes, this may happen because files are created in former days with wrong editor configurations. But on review, you see something like this. By the way, what is "*.map" good for?

Reply:
Yes. For example, the Less compiler generates source maps, which can be useful for debugging Less/CSS. I guess they're `.gitignore`d because they are only needed in development and can be easily regenerated.